src/openrct2/localisation/Convert.cpp

```diff
@@ -85,7 +85,7 @@ static std::string DecodeToMultiByte(const std::string_view& src)
 /**
  * Encodes a UTF-8 string as an RCT2 string.
  */
-static std::string Encode(const std::string_view& src)
+static std::string Encode(const std::string& src)
```

Comment:
Whats the reason behind the change here?

Reply:
Because `src` is treated as null-terminating string inside of `Encode` function.

Reply:
This encode function is not pretty but yeah I see the point, ideally it should not seek for the null terminator and rather use the range specified by string_view/string but thats for another PR.